package.json

```diff
@@ -16,7 +19,9 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "server": "json-server --watch db.json --port 5000",
```

Comment:
we'll be serving our local `json-server` from port `5000`, but letting [react proxy that port](https://github.com/MCRcodes/todo-demo/pull/1/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R3) and serve it using the default port used by [react: `3000`](https://github.com/MCRcodes/todo-demo/pull/1/files#diff-3d74dddefb6e35fbffe3c76ec0712d5c416352d9449e2fcc8210a9dee57dff67R10)